proto/types/interchain_da/da.proto

```diff
@@ -0,0 +1,27 @@
+// This file is a modified copy of the interchain-da module proto contract. Source:
+// https://github.com/dymensionxyz/interchain-da/blob/main/proto/dymension/interchain_da/da.proto.
+
+syntax = "proto3";
+package dymension.interchain_da;
+
+import "gogoproto/gogo.proto";
+import "types/cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/dymensionxyz/dymint/types/pb/interchain_da";
+
+message Params {
+  // CostPerByte defines the coin cost to store each byte of the blob. This
+  // allows for multidimensional gas pricing on different resources of the
+  // network.
+  cosmos.base.v1beta1.Coin cost_per_byte = 1 [ (gogoproto.nullable) = false ];
+  // MaxBlobSize is the hard cap of how many bytes a blob can be.
+  uint32 max_blob_size = 2;
+  // DisputePeriod is the number of blocks the blob is stored for.
+  uint64 dispute_period = 3;
```

Comment:
i would call it something like storage_window or prunning_height, since technically speaking is the time the blob will be stored regardless any dispute

Reply:
Makes sense @keruch , but you add what you just said as a comment too
thanks

Reply:
amazing ⭐ thanks